ccmlib/node.py

```diff
@@ -671,15 +671,15 @@ def stop(self, wait=True, wait_other_notice=False, other_nodes=None, gently=True
                 for node, mark in marks:
                     node.watch_log_for_death(self, from_mark=mark)
             else:
-                time.sleep(.1)
+                time.sleep(0.1)
 
             still_running = self.is_running()
             if still_running and wait:
                 # The sum of 7 sleeps starting at 1 and doubling each time
                 # is 2**7-1 (=127). So to sleep an arbitrary wait_seconds
                 # we need the first sleep to be wait_seconds/(2**7-1).
-                wait_time_sec = wait_seconds/(2**7-1.0)
-                for i in xrange(0, 7):
+                wait_time_sec = 0.1
+                while wait_time_sec <= wait_seconds + 0.1:
```

Comment:
we are not really forcing every iteration to 0.1 seconds in this case since wait_time_sec is growing ..... but fine